charmcraft/application/commands/store.py

```diff
@@ -1634,6 +1635,88 @@ def run(self, parsed_args):
             emit.message(cli.format_content(output_data, parsed_args.format))
 
 
+class FetchLibs(CharmcraftCommand):
+    """Fetch libraries defined in charmcraft.yaml."""
+
+    name = "fetch-libs"
+    help_msg = "Fetch one or more charm libraries"
+    overview = textwrap.dedent(
+        """
+        Fetch charm libraries defined in charmcraft.yaml.
+
+        For each library in the top-level `charm-libs` key, fetch the latest library
+        version matching those requirements.
+
+        For example:
+        charm-libs:
+          # Fetch lib with API version 0.
+          # If `fetch-libs` is run and a newer minor version is available,
+          # it will be fetched from the store.
+          - lib: postgresql.postgres_client
+            version: "0"
+          # Always fetch precisely version 0.57.
+          - lib: mysql.client
+            version: "0.57"
+        """
+    )
+    format_option = True
+    always_load_project = True
+
+    def run(self, parsed_args: argparse.Namespace) -> None:
+        """Fetch libraries."""
+        store = self._services.store
+        charm_libs = self._services.project.charm_libs
+        if not charm_libs:
+            raise errors.LibraryError(
```

Comment:
If there aren't any libs defined in `charmcraft.yaml`, the user probably meant to use `fetch-lib`. However, I'm not specifically advertising `fetch-lib` because defining the libs in the charmcraft.yaml file seems to be the way forward.